### DISCUSSION TITLE
Variable "form_search_in_agency" does not exist. 500 Internal Server Error - Twig_Error_Runtime

### DISCUSSION THREAD

**Discussion by Ronnal-Zp** [symfony/symfony#47559] [Q&A]
Hello, I am having this error in my .twig template, it indicates that the variable " form_search_in_agency" does not exist, I do not understand where the variable " form_search_in_agency" comes from, I show you the "return " of the controller

```php
 $parameters = array(
                'result' => $properties,
                'search' => $search,
                'form' => $form->createView(),
                'type' => 'list',
                'agencies' => $agencies,
                'attrslug' => null,
                'minprice' => $minprice,
                'favorites' => $favorites,
                'routes' => $routes,
                'nearproperties' => isset($nearproperties) ? $nearproperties : null
            );

$response = $this->render(':main/search:list.html.twig', $parameters);
return $response;
```

Template twig where the error jumps
```twig
{% if form_search_in_agency %}
                <div class="orderbar">
                    <div class="container pa0">
                    {{ form_start(form_search_in_agency) }}
                    {{ form_errors(form_search_in_agency) }}
```

**Comment by stof** [comment]
Well, you have a `{% if form_search_in_agency %}` which tries to use a `form_search_in_agency` variable. And this variable does not exist (your controller defines a `form` variable, not `form_search_in_agency`)

**Reply by Ronnal-Zp** [reply]
> 

does it mean that "form" would be equivalent to "form_search_in_agency"?, this is a new project that I have been assigned and I do not have much knowledge in symfony, sorry, but if it were so, I should find it correctly right?

**Reply by stof** [reply]
I have no idea. It depends what the dev creating that template expected when they wrote code using a `form_search_in_agency` variable. Just knowing that variable name does not give me the knowledge of the context of that code.

**Reply by Ronnal-Zp** [reply]
> I have no idea. It depends what the dev creating that template expected when they wrote code using a `form_search_in_agency` variable. Just knowing that variable name does not give me the knowledge of the context of that code.

Well, thank you very much for answering me, I will continue to see what else it can be
